### DISCUSSION TITLE
Will there be more support for Chinese characters?

### DISCUSSION THREAD

**Discussion by Ricoreo** [witalihirsch/qBitTorrent-fluent-theme#53] [Ideas]
Love this theme and it looks great in Windows 11, but the fonts in the app looks weird when using Chinese version of Windows, with no smooth font effect. I suppose it's because the default font in CHN Windows is different? Is there a chance to use Microsoft Yahei (a default font in CHN Windows) or Segoe UI (IDK if this will work for Chinese) as the defaullt font?

The font in the original theme looks better.
![original](https://github.com/witalihirsch/qBitTorrent-fluent-theme/assets/10759598/81a52fac-95fe-4558-acbf-cd96e3ae661b)
![fluent](https://github.com/witalihirsch/qBitTorrent-fluent-theme/assets/10759598/c92b9779-7429-42aa-ab4e-ed026dda8110)

**Comment by dyphire** [comment]
Implementing it only requires optimizing font fallback for CJK: https://github.com/dyphire/qBitTorrent-fluent-theme/commit/6f1b3f7d33328f7ec431497cbe6fedfcacb5c9ef
![image](https://github.com/witalihirsch/qBitTorrent-fluent-theme/assets/61936050/a7599e43-d6bf-4fb1-9e68-4e92a1f49cf2)
You can download these theme there: https://github.com/dyphire/qBitTorrent-fluent-theme/releases

**Reply by 351492384fate** [reply]
Sorry, the link is returning 404 . Could you please provide a new

**Reply by 351492384fate** [reply]
How should this be resolved specifically?

**Reply by dyphire** [reply]
> How should this be resolved specifically?

The changes have long been merged into this repository: #55
Since the release hasn't been updated for a long time, you can just download the theme folder of the main branch: [themes](https://github.com/witalihirsch/qBitTorrent-fluent-theme/tree/main/themes)

Edit: This is an archive of the original files.
[Theme.zip](https://github.com/user-attachments/files/18220689/Theme.zip)
